sdk/communication/communication-network-traversal/review/communication-network-traversal.api.md

```diff
@@ -25,14 +32,7 @@ export interface CommunicationRelayClientOptions extends PipelineOptions {
 // @public
 export interface CommunicationRelayConfiguration {
     expiresOn: Date;
-    turnServers: CommunicationTurnServer[];
-}
-
-// @public
-export interface CommunicationTurnServer {
-    credential: string;
-    urls: string[];
-    username: string;
+    iceServers: CommunicationIceServer[];
```

Comment:
Can we list the renames as breaking changes in the Changelog.md file to help developers with the update?